pyproject.toml

```diff
@@ -7,28 +7,44 @@ requires-python = ">=3.10"
 dependencies = [
     "einops>=0.8.0",
     "huggingface_hub>=0.26.0",
-    "ipykernel>=6.29.5",
-    "ipywidgets>=8.1.7",
-    "numpy>=1.24.0",
+    "numpy>=2.0.0",
```

Comment:
Why are we changing the dependencies?

Reply:
oops I cherrypicked the pyproject.toml from the original PR, looks like there's also some unintended dependency diffs. I'll revert this.